thirdeye/thirdeye-pinot/src/main/resources/com/linkedin/thirdeye/detector/multiple-anomalies-email-template.ftl

```diff
@@ -93,7 +93,7 @@
         <#if cid?has_content>
           <tr>
               <td style="padding: 24px;" colspan="2" align="center">
-                <img style="width: 70%;" src="cid:${cid}"\>
+                <a href="${anomalyDetails[0].anomalyURL}${anomalyDetails[0].anomalyId}" target="_blank"><img style="width: 70%;" src="cid:${cid}"\></a>
```

Comment:
any reason you're not doing a `link-to` here instead of the `a` tag?

Reply:
it's not in ember. This is a free marker file used from the java Email Resources.